docs/security.rst

```diff
@@ -28,8 +28,8 @@ Running containers with arbitrary user IDs
   The images can be run with arbitrary user IDs. If there is a container engine vulnerability, arbitrary user IDs can prevent processes from escaping the container and gaining escalated permissions on the host node.
 
 
-For details on the available security features in Confluent platform, see the `Confluent Platform Security
-Overview Documentation <http://docs.confluent.io/current/kafka/security.html>`_.
+For details on the available security features in Confluent platform, see the :ref:`Confluent Platform Security
+Overview Documentation <security>`.
 
-For tutorials on using SSL in the Confluent Platform, see the documented tutorials on `SSL
-<http://docs.confluent.io/current/kafka/ssl.html>`_ and `SASL <http://docs.confluent.io/current/kafka/sasl.html>`_.
+For tutorials on using SSL in the Confluent Platform, see the documented tutorials on :ref:`SSL
+<security>` and :ref:`SASL <security>`.
```

Comment:
SSL could point to `kafka_ssl_authentication` and SASL could point to `kafka_sasl_auth`